### PR TITLE
Added aux kernels for specific enthalpy and stagnation pressure and t…

### LINKED ISSUE #7975: Create aux kernels for specific enthalpy and stagnation pressure and temperature
### Description of the enhancement or error report
Add 3 aux kernels for `fluid_properties` module:

- `SpecificEnthalpyAux`
- `StagnationPressureAux`
- `StagnationTemperatureAux`

### Rationale for the enhancement or information for reproducing the error
These aux kernels are currently not required by any other object but can be useful in analysis of output for certain fluid problems.

### Identified impact
These changes would have no impact on any interface; they just provide additional aux kernels.

### PR DESCRIPTION
**Pull request by joshuahansel**
Added aux kernels for the following quantities to the fluid properties module:
 - specific enthalpy (not total enthalpy)
 - stagnation pressure
 - stagnation temperature

Closes #7975

### CODE REVIEW

File: modules/fluid_properties/src/auxkernels/StagnationTemperatureAux.C [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+StagnationTemperatureAux::StagnationTemperatureAux(const InputParameters & parameters) :
+    AuxKernel(parameters),
+    _specific_volume(coupledValue("v")),
+    _specific_internal_energy(coupledValue("e")),
+    _velocity(coupledValue("vel")),
+    _fp(getUserObject<SinglePhaseFluidProperties>("fp"))
+{
+}
+
+StagnationTemperatureAux::~StagnationTemperatureAux()
```
Comment (dschwen) [inline modules/fluid_properties/src/auxkernels/StagnationTemperatureAux.C]: none of these empty virtual destructors is necessary

**Comment by joshuahansel** [comment]
I cannot view why the "App tests:linux-gnu" check failed - CIVET says

> You are not allowed to see results. Please sign in to an appropiate user.

even though I am signed in, so I must be an inappropriate user...

**Comment by andrsd** [comment]
Building bighorn failed, because it contains kernels with the same names. Those kernel are obsolete now, so I will remove them. Bighorn should be using the ones you wrote...

**Comment by permcody** [comment]
@joshuahansel have @brianmoose add you to the "collaborators" group if you are trying to look at the target for internal applications.

**Review by andrsd — APPROVED** [review]

**Pull request merged**